assignments/closure.js

```diff
@@ -1,21 +1,43 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+const goodbye = 'Thanks for all the fish!';
+function printGoodbye() { return goodbye; }
+console.log(printGoodbye());
+
 
 /* STRETCH PROBLEMS, Do not attempt until you have completed all previous tasks for today's project files */
 
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
   // Return a function that when invoked increments and returns a counter variable.
+  let runningCount = 0;
+  return (function() { return ++runningCount });
```

Comment:
remember you can use your ES6 syntax to make these types of functions more concise

```
const counter = () => {
  let runningCount = 0;
  return () => ++runningCount;
}
```